cirq/testing/logs_test.py

```diff
@@ -118,11 +118,18 @@ def test_assert_logs_log_level():
         logging.error('orange apple fruit')
         logging.debug('should not')
         logging.info('count')
-    with cirq.testing.assert_logs('apple', level=logging.INFO, count=2):
+    with cirq.testing.assert_logs('apple', min_level=logging.INFO, count=2):
         logging.error('orange apple fruit')
         logging.debug('should not')
         logging.info('count')
 
+    with cirq.testing.assert_logs('info only 1', min_level=logging.INFO, max_level=logging.INFO):
```

Comment:
As written, this test will pass even if `max_level` is ignored: `info only 1` appears in a log of level `INFO` or greater, and `count` is unspecified and therefore not checked.

Reply:
```
>       assert count is None or len(records) == count, (
            f'Expected {count} log message but ' f'got {len(records)}. Log messages: ' f'{msgs}'
        )
E       AssertionError: Expected 1 log message but got 2. Log messages: ['info only 1', 'info warning 1']
```